src/resource/archive/O2rArchive.h

```diff
@@ -1,5 +1,4 @@
 #pragma once
-
 #undef _DLL
```

Comment:
just to keep the diff clean we should put that newline back

also (def out of scope for this PR) we should make an issue to clean up `#pragma once` use and standardize on `#ifndef` based header guards